newsletters/2023-07-19-newsletter.md

```diff
@@ -0,0 +1,69 @@
+* [Cycles of curves: what are they and do we need them?](https://blog.icme.io/can-we-avoid-cycles-of-curves/) - article ny Wyatt Benno. (13.07.2023)
+* [Understanding ZK for Beginners: Simplified Explanation with Examples](https://medium.com/@santiagotrujilloz/understanding-zk-for-beginners-simplified-explanation-with-examples-a98182fd5ffb) - article by Santiago Trujillo Zuluaga. (13.07.2023)
+* [Hybrid Rollup — A Bird’s Eye View](https://medium.com/@ProjectZKM/hybrid-rollup-a9908f3ee69f) - article by ZKM. (14.07.2023)
+* [What Are Rollups In Crypto? ZK- Rollups & Optimistic Rollups Explained](https://preciousruby.medium.com/what-are-rollups-in-crypto-zk-rollups-optimistic-rollups-explained-c2808681d12c) - article by PreciousRuby Okeke. (14.07.2023)
+* [Zero-Knowledge Proofs Unveiled: From Theory to Practice with Aleo. Chapter 6](https://web3illy.medium.com/zero-knowledge-proofs-unveiled-from-theory-to-practice-with-aleo-5d803ac90c0d) and [Chapter 7](https://web3illy.medium.com/zero-knowledge-proofs-unveiled-from-theory-to-practice-with-aleo-fc263603598b) - another chapter from the series of articles by Illy’s Web3 blog. (15.07.2023)
+* [Zero Knowledge Proofs: Empowering the Future of Data Privacy and Scalable Infrastructure, Part 1](https://medium.com/@syedarmani/zero-knowledge-proofs-empowering-the-future-of-data-privacy-and-scalable-infrastructure-part-1-87ee6ed7ed36) - article by Syed Armani. (17.07.2023)
+* [zkML: Unleashing the Future of Collectibles](https://onchainguy.medium.com/zkml-unleashing-the-future-of-collectibles-7619f137c4e7) article by Sebastian. (17.07.2023)
+* [Moving the Real World to web3 With zk-Oracle](https://medium.com/@talhauns96/moving-the-real-world-to-web3-with-zk-oracle-675517bdbcfb) - article by talhauns96. (18.07.2023)
+
+### ZK projects
+* [Hyper Oracle, Simply Explained](https://mirror.xyz/hyperoracleblog.eth/FKvpIGI7fiuNr5rnTlFWAdxk4QCNFIR9rmqDPxWLc2w). (12.07.2023)
+* [Implementing Coordinated Vulnerability Disclosures at Dusk](https://dusk.network/news/implementing-cvd-at-dusk). (12.07.2023)
+* [Polygon 2.0: Tokenomics](https://polygon.technology/blog/polygon-2-0-tokenomics). (13.07.2023)
+* [Contribute to Scroll](https://scroll.io/blog/contributeToScroll). (13.07.2023)
+* [Scroll - Weekly Update](https://twitter.com/Scroll_ZKP/status/1679962721650118656). (14.07.2023)
+* [The Horizen Q2 2023 Report](https://blog.horizen.io/the-horizen-q2-2023-report/?utm_source=social&utm_medium=social&utm_campaign=Marketing). (14.07.2023)
+* [Taiko - Announcing our first community grant program](https://taiko.mirror.xyz/G7dmuoR42S4D55vT8bs_lAxPZP63kAgRu2IfqkJdf6U). (14.07.2023)
+* [Bridging on Mantle Mainnet](https://www.mantle.xyz/blog/announcements/bridging-on-mantle-mainnet). (17.07.2023)
+* [zkSync launches new proof system called Boojum for Era mainnet](https://www.theblock.co/post/239880/zksync-launches-new-proof-system-called-boojum-for-era-mainnet). (17.07.2023)
```

Comment:
Better link to the source: https://twitter.com/zksync/status/1680820952991408129